### DISCUSSION TITLE
My new page

### DISCUSSION THREAD

**Discussion by shamameer** [community/community#45468] [Other Feature Feedback, Questions, & Ideas] [answered]
Eror 404 on y new page

**Comment by radar-blip** [comment, marked as answer]
This isn't going to work due to the spaces `<link rel="stylesheet" href="/storage/emulated/0/sham/my project sham.css/">`
You will need to change `my project sham.css` in both your html code, and in your repository. (it's common to just use `style.css`)
And since your link to .css is in same directory, all you need is: `<link rel="stylesheet" href="style.css/">`

Also, would probably be a good idea to change Index.html to _lowercase_ index.html in order for GitHub Pages to recognize it.

**Reply by shamameer** [reply]
Thank you
Ad photo it s not visubal
